docs/docs/image-tutorial.md

```diff
@@ -0,0 +1,271 @@
+   /*
+    * Gatsby's data processing layer begins with “source”
+    * plugins. Here the site sources its data from Wordpress.
+    */
+   {
+     resolve: `gatsby-source-wordpress`,
+     options: {
+       /*
+       * The base URL of the Wordpress site without the trailing slash and the protocol. This is required.
+       * Example : 'gatsbyjswpexample.wordpress.com' or 'www.example-site.com'
+       */
+       baseUrl: `dev-gatbsyjswp.pantheonsite.io`,
+       // The protocol. This can be http or https.
+       protocol: `http`,
+       // Indicates whether the site is hosted on wordpress.com.
+       // If false, then the asumption is made that the site is self hosted.
+       // If true, then the plugin will source its content on wordpress.com using the JSON REST API V2.
+       // If your site is hosted on wordpress.org, then set this to false.
+       hostingWPCOM: false,
+       // If useACF is true, then the source plugin will try to import the Wordpress ACF Plugin contents.
+       // This feature is untested for sites hosted on Wordpress.com
+       useACF: true,
+     },
+   },
+   'gatsby-plugin-react-helmet',
+   'gatsby-transformer-sharp',
+   'gatsby-plugin-sharp',
+   {
+     resolve: `gatsby-source-filesystem`,
+     options: {
+       name: `images`,
+       path: `${__dirname}/src/images/`,
+     },
+   },
+ ],
+}
+```
+
+### Creating GraphQL queries that pull in images from WordPress
+
+Now you are ready to create a GraphQL query to pull in some images from the WordPress site before we can view them in your browser. You will create a query that pulls in the images.
+
+Run:
+
+```shell
+`gatsby develop`
+```
+
+Open localhost:8000 and localhost:8000/__graphql. 
+
+Here’s an example of creating specific widths and heights for images:
+
+```jsx
+{
+  allWordpressPost {
+    edges {
+      node {
+        childWordPressAcfPostPhoto {
+          photo {
+            localFile {
+              childImageSharp {
+                # edit the maxWidth value to generate resized
+                # images
+                resolutions(width: 200, height: 200) {
+                  # use field names in the GraphQL explorer
+                  # in your site, comment them out and use
+                  # the fragments provided by Gatsby instead
+                  src
+                  
+                  # This fragment won't work in the GraphQL
+                  # explorer, but you can use it in your site
+                  # ...GatsbyImageSharpResolutions_withWebp
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+
+Here’s an example query for generating different sizes of an image:
+
+```jsx
+{
+  allWordpressPost {
+    edges {
+      node {
+        childWordPressAcfPostPhoto {
+          photo {
+            localFile {
+              childImageSharp {
+                # edit the maxWidth value to generate resized
```

Comment:
Similar changes as above, something like this?

`# Try editing the "maxWidth" value to generate resized images`